osh/hub/osh_xmlrpc/worker.py

```diff
@@ -76,6 +79,27 @@ def finish_task(request, task_id):
                 task.fail_task()
 
 
+@validate_worker
+def open_task(request, task_id):
+    response = kobo_open_task(request, task_id)
+
+    if settings.ENABLE_SINGLE_USE_WORKERS:
```

Comment:
Will this work when `ENABLE_SINGLE_USE_WORKERS` is missing in `settings_local.py`?

Reply:
No, it would throw `'Settings' object has no attribute` exception.

Reply:
Then it should be fine.  Btw. it seems you forgot to resolve this TODO before clearing the draft flag on this PR: https://github.com/openscanhub/openscanhub/pull/234/files#diff-e1475eca3773eef4cb4c8503a87591c770a7ed2ebf8e08b205c00faae6ab81a2R187

Reply:
I just added it today and need to add it to https://pagure.io/fedora-infra/ansible/blob/main/f/roles/openshift-apps/openscanhub/templates/settings_local.py

Reply:
https://pagure.io/fedora-infra/ansible/pull-request/1884